tests/include/test/macros.h

```diff
@@ -61,6 +61,16 @@
         }                                                    \
     } while (0)
 
+/** This macro asserts fails the test with given output message.
+ *
+ * \param   MESSAGE The message to be outputed on assertion
+ */
+#define TEST_FAIL(MESSAGE)                           \
+    do {                                                  \
+        mbedtls_test_fail(MESSAGE, __LINE__, __FILE__);   \
+        goto exit;                                          \
+    } while (0)                                              \
+
```

Comment:
(minor) for new code, and editing blocks of existing code, can we line up the `\`s on the right? I normally aim for one or two spaces from the rightmost line ending (i.e. would be two spaces from the `;` on the `mbedtls_test_fail()` line)

Reply:
Other than that this is looking good

Reply:
oh - also remove the trailing `\` on the `} while (0)` line, please

Reply:
(minor) the trailing `\` on line 68 doesn't line up, but only make that change if another change is needed later